app/tests/test_search.py

```diff
@@ -211,29 +231,81 @@ def test_search_results_summary_top_search(
             == "Search by file name, transferring body, series or consignment reference."
         )
 
+    @patch("app.main.routes.OpenSearch")
     def test_search_results_summary_no_query(
-        self, client: FlaskClient, mock_all_access_user
+        self, mock_search_client, client: FlaskClient, mock_all_access_user
     ):
         """
         Given an all_access_user accessing the search results summary page
         When they make a GET request without a query
         Then they should not see any results on the page.
         """
+        mock_search_client.return_value = MockOpenSearch(
```

Comment:
I think this mock is not necessary for this test, as this is the scenario where there is no query and so opensearch.query is not called.